src/react-apps/utils/convertLangToJson.ts

```diff
@@ -0,0 +1,6 @@
+// Is to be executed with node in terminal to convert language
+const fs = require('fs');
+const ini = require('ini');
+
+const obj = ini.decode(fs.readFileSync('./../../../../../AltinnCore/Common/Languages/ini/nb.ini', 'utf-8'));
+console.log(obj);
```

Comment:
Is this the whole file? No exported functions or consts?

Reply:
Yes, just copy from the output. Easiest there is. Just converting without creating a file. We're importing the result in another ts export.